model/otel/metrics.yaml

```
@@ -21,9 +22,12 @@ groups:
     unit: "{span}"
     note: |
       For spans with `recording=true`: Implementations MUST record both `otel.sdk.span.live` and `otel.sdk.span.ended`.
-      For spans with `recording=false`: If implementations decide to record this metric, they MUST also record `otel.sdk.span.live`.
+      For spans with `recording=false`: If implementations decide to not record `otel.sdk.span.live`, they MUST record `otel.sdk.span.ended` on span start for those spans instead.
+      In other words, for spans with `recording=false` implementations may treat them as ended directly after they are started from a metrics perspective.
+      This ensures that `otel.sdk.span.ended` is guaranteed to provide insights about the total span counts and sampling rates, even if spans are not recorded.
```

```suggestion
      Spans with `recording=false` are considered ended immediately after Start to ensure SDKs do not need to track state for non-recording spans.
```
I would love to simplify the wording here, but you don't need to take this verbatim.